site/src/pages/WorkspaceChangeVersionPage/WorkspaceChangeVersionForm.tsx

```diff
@@ -0,0 +1,155 @@
+  const styles = useStyles()
+  const { t } = useTranslation("workspaceChangeVersionPage")
+  const formik = useFormik({
+    initialValues: {
+      versionId: workspace.latest_build.template_version_id,
+    },
+    validationSchema,
+    onSubmit: ({ versionId }) => onSubmit(versionId),
+  })
+  const autocompleteValue = versions.find(
+    (version) => version.id === formik.values.versionId,
+  )
+
+  return (
+    <form onSubmit={formik.handleSubmit}>
+      <Stack direction="column" spacing={3}>
+        <Stack
+          direction="row"
+          spacing={2}
+          className={styles.workspace}
+          alignItems="center"
+        >
+          <div className={styles.workspaceIcon}>
+            <img src={workspace.template_icon} alt="" />
+          </div>
+          <Stack direction="column" spacing={0.5}>
+            <span className={styles.workspaceName}>{workspace.name}</span>
+
+            <span className={styles.workspaceDescription}>
+              {workspace.template_display_name.length > 0
+                ? workspace.template_display_name
+                : workspace.template_name}
+            </span>
+          </Stack>
+        </Stack>
+
+        <Autocomplete
+          id="workspaceVersion"
+          disableClearable
+          options={versions.slice().reverse()}
+          value={autocompleteValue}
+          onChange={async (_event, value) => {
+            if (value) {
+              await formik.setFieldValue("versionId", value.id)
+            }
+          }}
+          renderInput={(params) => (
+            <TextField
+              {...params}
+              label={t("labels.workspaceVersion")}
+              variant="outlined"
+              fullWidth
+            />
+          )}
+          getOptionLabel={(version: TemplateVersion) => version.name}
+          renderOption={(version: TemplateVersion) => (
+            <div className={styles.menuItem}>
+              <div>
+                <div>{version.name}</div>
+                <div className={styles.versionDescription}>
+                  {t("labels.createdBy")} {version.created_by.username}{" "}
+                  {createDayString(version.created_at)}
+                </div>
+              </div>
+
+              {template.active_version_id === version.id && (
+                <Pill
+                  type="success"
+                  text={t("labels.active")}
+                  className={styles.activePill}
+                />
+              )}
+            </div>
+          )}
+        />
+      </Stack>
+
+      <FormFooter
+        onCancel={onCancel}
+        isLoading={isLoading}
+        submitLabel={t("labels.submit")}
+      />
+    </form>
+  )
+}
+
+const useStyles = makeStyles((theme) => ({
+  workspace: {
+    padding: theme.spacing(2.5, 3),
+    borderRadius: theme.shape.borderRadius,
+    backgroundColor: theme.palette.background.paper,
+    border: `1px solid ${theme.palette.divider}`,
+  },
+
+  workspaceName: {
+    fontSize: 16,
```

Comment:
wonder if you can use the theme for any of these font sizes

Reply:
Sure thing.